data/cncf.yaml

```diff
@@ -294,26 +330,32 @@
       url: https://github.com/cloudevents/spec
       check_sets:
         - community
+        - docs
     - name: sdk-go
       url: https://github.com/cloudevents/sdk-go
       check_sets:
         - code-lite
+        - docs
     - name: sdk-javascript
       url: https://github.com/cloudevents/sdk-javascript
       check_sets:
         - code-lite
+        - docs
     - name: sdk-csharp
       url: https://github.com/cloudevents/sdk-csharp
       check_sets:
         - code-lite
+        - docs
```

Comment:
The `docs` check set is a subset of the `code-lite` check set, so it isn't needed in this case. Please remove it.